src/components/adjudications/AdjudicationDialog.jsx

```diff
@@ -131,7 +144,8 @@ AdjudicationDialog.propTypes = {
     judgeName: PropTypes.string,
     specialAward: PropTypes.bool,
     technicalMark: PropTypes.number
-  }).isRequired
+  }).isRequired,
+  performanceValues: PropTypes.shape().isRequired
```

Comment:
Typically when we add in a `PropTypes.shape()`, we also pass in a structure to assure that our shape is valid.

Ex: https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes Look at optionalObjectWithShape